README.md

```diff
@@ -50,116 +50,83 @@ Stay up to date with the latest ONNX news. [[Facebook](https://www.facebook.com/
 
 # Installation
 
-## Binaries
-
-A binary build of ONNX is available from [Conda](https://conda.io), in [conda-forge](https://conda-forge.org/):
-
+## Official Python packages
+ONNX released packages are published in PyPi.
 ```
-conda install -c conda-forge onnx
+pip install numpy protobuf==3.11.3
+pip install onnx
 ```
 
-## Source
-
-If you have installed onnx on your machine, please `pip uninstall onnx` first before the following process of build from source.
+[Weekly packages](https://test.pypi.org/project/onnx-weekly/) are published in test pypi to enable experimentation and early testing.
 
-### Linux and MacOS
-You will need an install of Protobuf and NumPy to build ONNX.  One easy
-way to get these dependencies is via
-[Anaconda](https://www.anaconda.com/download/):
 
+## Conda packages
+A binary build of ONNX is available from [Conda](https://conda.io), in [conda-forge](https://conda-forge.org/):
 ```
-# Use conda-forge protobuf, as default doesn't come with protoc
-conda install -c conda-forge protobuf numpy
+conda install -c conda-forge numpy protobuf==3.11.3 libprotobuf=3.11.3
```

Comment:
I can confirm that version 3.12.3 works as well (though we build it rather than installing from conda)

Reply:
using 3.11.3 as we use this version in release and CI pipelines